README.md

```diff
@@ -31,15 +31,15 @@ repositories {
 }
 
 dependencies {
-    compile "io.projectreactor:reactor-core:3.5.7"
-    testCompile "io.projectreactor:reactor-test:3.5.7"
+    compile "io.projectreactor:reactor-core:3.6.0-SNAPSHOT"
+    testCompile "io.projectreactor:reactor-test:3.6.0-SNAPSHOT"
 
     // Alternatively, use the following for latest snapshot artifacts in this line
-    // compile "io.projectreactor:reactor-core:3.5.8-SNAPSHOT"
-    // testCompile "io.projectreactor:reactor-test:3.5.8-SNAPSHOT"
+    // compile "io.projectreactor:reactor-core:3.6.0-SNAPSHOT"
```

Comment:
I think we can instead suggest "trying the upcoming release line" and use such wording.

Reply:
reverted